packages/web/src/common/store/notifications/fetchNotifications.ts

```diff
@@ -111,58 +85,19 @@ function* fetchDiscoveryNotifications(params: FetchNotificationsParams) {
     'claimable_reward'
   ].filter(removeNullable)
 
-  const discoveryNotifications = yield* call(
-    audiusBackendInstance.getDiscoveryNotifications,
+  const { data } = yield* call(
+    [sdk.full.notifications, sdk.full.notifications.getNotifications],
     {
       timestamp: timeOffset,
-      groupIdOffset,
-      limit,
-      validTypes
+      groupId: groupIdOffset,
+      validTypes,
+      userId: encodedUserId,
+      limit
     }
   )
+  const notifications = data
+    ? transformAndCleanList(data.notifications, notificationFromSDK)
+    : []
```

Comment:
Dare I ask if you used `compareSDKResponse` to check if the sdk version matches the audiusBackend version? 😅